From 9a4cdd413fd51f86c78897c88980b65cd8413b75 Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Date: Tue, 2 Aug 2011 19:42:12 +0000
Subject: [PATCH] staging: rtl8192u: declare MODULE_FIRMWARE

declaring MODULE_FIRMWARE has apparently forgotten while removing the embedded
firmware arrays in 0a8692b534e18fcec6eac07551bb37a22659f5c7 (Staging: rtl8192u:
remove a copy of 80211.h).

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
Cc: stable <stable@kernel.org> [2.6.39+]
---
 drivers/staging/rtl8192u/r819xU_firmware.c |    9 +++------
 1 files changed, 3 insertions(+), 6 deletions(-)

--- a/drivers/staging/rtl8192u/r819xU_firmware.c
+++ b/drivers/staging/rtl8192u/r819xU_firmware.c
@@ -399,10 +399,7 @@ download_firmware_fail:
 
 }
 
-
-
-
-
-
-
+MODULE_FIRMWARE("RTL8192U/boot.img");
+MODULE_FIRMWARE("RTL8192U/main.img");
+MODULE_FIRMWARE("RTL8192U/data.img");
 
